tests/DynamoDbSessionTest.php

```diff
@@ -2,15 +2,41 @@
 
 namespace SilverStripe\DynamoDb\Tests;
 
+use ReflectionProperty;
+use SilverStripe\Core\Environment;
 use SilverStripe\Dev\SapphireTest;
 use SilverStripe\DynamoDb\Model\DynamoDbSession;
 
 class DynamoDbSessionTest extends SapphireTest
 {
-
     public function testGetReturnsNullWhenNotConfigured()
     {
+        // Ensure the session table is null for this test
+        $origValue = Environment::getEnv('AWS_DYNAMODB_SESSION_TABLE');
+        Environment::setEnv('AWS_DYNAMODB_SESSION_TABLE', null);
+
         $this->assertNull(DynamoDbSession::get());
+
+        Environment::setEnv('AWS_DYNAMODB_SESSION_TABLE', $origValue);
+    }
+
+    public function testGetReturnsSessionWhenConfigured()
```

Comment:
Added this because the other test could be satisfied by simply always returning null - which is obviously not correct.